autorest/codegen/serializers/builder_serializer.py

```diff
@@ -860,6 +878,8 @@ def _call_request_builder_helper(
         if not self.code_model.options["version_tolerant"]:
             template_url = template_url or f"self.{builder.name}.metadata['url']"
             retval.append(f"    template_url={template_url},")
+        retval.append('    headers=_headers,')
+        retval.append('    params=_params,')
```

Comment:
can you also update line `1015` to
`retval.append("error_map.update(kwargs.pop('error_map', {})) or {}")`